spelling_wordlist.txt

```diff
@@ -159,4 +160,4 @@ verifier
 verifiers
 walkthrough
 whitespace
-prover_args
\ No newline at end of file
+prover_args
```

Comment:
This should not be here; instead it should be surrounded by backticks wherever it is used.

Reply:
@mdgeorge4153 I did not add this. Gambit / certoraMutate doesn't have any such argument. No idea why it shows up in the diff and who is using it.